Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Kimai - Time tracking/Main application](https://hosted.weblate.org/projects/kimai/messages/).


It also includes following components:

* [Kimai - Time tracking/Teams](https://hosted.weblate.org/projects/kimai/teams/)

* [Kimai - Time tracking/Wizard](https://hosted.weblate.org/projects/kimai/wizard/)

* [Kimai - Time tracking/Emails](https://hosted.weblate.org/projects/kimai/emails/)

* [Kimai - Time tracking/Export](https://hosted.weblate.org/projects/kimai/export/)

* [Kimai - Time tracking/User feedback (Flash messages and Alerts)](https://hosted.weblate.org/projects/kimai/flashmessages/)

* [Kimai - Time tracking/Invoice-Numbers](https://hosted.weblate.org/projects/kimai/invoice-numbers/)

* [Kimai - Time tracking/Invoice-Templates](https://hosted.weblate.org/projects/kimai/invoice-templates/)

* [Kimai - Time tracking/Actions (Dropdowns and Toolbars)](https://hosted.weblate.org/projects/kimai/actions/)

* [Kimai - Time tracking/Plugins](https://hosted.weblate.org/projects/kimai/plugins/)

* [Kimai - Time tracking/Reporting](https://hosted.weblate.org/projects/kimai/reporting/)

* [Kimai - Time tracking/Invoice-Calculator](https://hosted.weblate.org/projects/kimai/invoice-calculator/)

* [Kimai - Time tracking/Validations](https://hosted.weblate.org/projects/kimai/validations/)

* [Kimai - Time tracking/Date-Picker](https://hosted.weblate.org/projects/kimai/daterangepicker/)

* [Kimai - Time tracking/System configurations](https://hosted.weblate.org/projects/kimai/system-configuration/)

* [Kimai - Time tracking/About](https://hosted.weblate.org/projects/kimai/about/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/kimai/-/messages/horizontal-auto.svg)

kevinpapst commented on translations/messages.pt_BR.xlf:
```diff
@@ -97,7 +97,7 @@
       <trans-unit id="ebTjBb9" resname="my_times">
         <source>my_times</source>
-        <target>Meus tempos</target>
+        <target state="translated">Meus horários</target>
```
@wtuemura are you a Kimai user? I saw these changes (thank you for helping out with the translations 👍), as you replaced "de trabalho" with "trabalhadas" below in several keys and and I became curious. 

I don't speak Portuguese, so I tried G-Translate and "Meus tempos" does match the expected translation, while "Meus horários" translates to "my schedules", which would be wrong. 

That's why I am asking if you are a Kimai user => some translations are obviously much harder to understand without context. The screen "My times" opens a list of all my logged working times and not something like scheduled calendar entries.

These changes are currently blocking a merge and I would like to resolve that issue. Would be great if you could give me your opinion on that.

kevinpapst replied:
Clarified at hosted.weblate.org

weblate committed:
Translated using Weblate (Polish)

Currently translated at 100.0% (75 of 75 strings)

Translated using Weblate (Polish)

Currently translated at 96.2% (332 of 345 strings)

Translated using Weblate (Italian)

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Romanian)

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Romanian)

Currently translated at 50.0% (2 of 4 strings)

Translated using Weblate (German)

Currently translated at 75.0% (3 of 4 strings)

Translated using Weblate (English)

Currently translated at 100.0% (4 of 4 strings)

Translated using Weblate (Persian)

Currently translated at 100.0% (345 of 345 strings)

Translated using Weblate (Arabic)

Currently translated at 100.0% (7 of 7 strings)

Translated using Weblate (Arabic)

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Arabic)

Currently translated at 100.0% (9 of 9 strings)

Translated using Weblate (Arabic)

Currently translated at 96.0% (72 of 75 strings)

Translated using Weblate (Arabic)

Currently translated at 94.2% (325 of 345 strings)

Added translation using Weblate (Arabic)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (345 of 345 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (345 of 345 strings)

Update translation files

Updated by "Cleanup translation files" hook in Weblate.

Co-authored-by: ButterflyOfFire <boffire@users.noreply.hosted.weblate.org>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kevin Papst <kevin@kevinpapst.de>
Co-authored-by: Massimo Pissarello <mapi68@gmail.com>
Co-authored-by: Roya K <royakhajavi@gmail.com>
Co-authored-by: Simona Iacob <s@zp1.net>
Co-authored-by: Wellington Terumi Uemura <wellingtonuemura@gmail.com>
Co-authored-by: gnu-ewm <gnu.ewm@protonmail.com>
Translate-URL: https://hosted.weblate.org/projects/kimai/about/de/
Translate-URL: https://hosted.weblate.org/projects/kimai/about/en/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-calculator/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-templates/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-templates/ro/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/fa/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/it/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/pl/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/kimai/plugins/ro/
Translate-URL: https://hosted.weblate.org/projects/kimai/system-configuration/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/system-configuration/pl/
Translate-URL: https://hosted.weblate.org/projects/kimai/wizard/ar/
Translation: Kimai - Time tracking/About
Translation: Kimai - Time tracking/Invoice-Calculator
Translation: Kimai - Time tracking/Invoice-Templates
Translation: Kimai - Time tracking/Main application
Translation: Kimai - Time tracking/Plugins
Translation: Kimai - Time tracking/System configurations
Translation: Kimai - Time tracking/Wizard
